alacritty/src/display/window.rs

```diff
@@ -147,7 +148,12 @@ impl Window {
         }
 
         #[cfg(not(any(target_os = "macos", windows)))]
-        if let Some(token) = event_loop.read_token_from_env() {
+        if let Some(token) = _options
+            .activation_token
+            .take()
+            .map(ActivationToken::from_raw)
+            .or_else(|| event_loop.read_token_from_env())
```

Comment:
I think there's just a `.map_or_else` function?

Reply:
It works differently, I'd need to manually wrap in `Some` the result of building a token, so it'll be more verbose.